src/com/mendix/widget/carousel/ui/Carousel.css

```diff
@@ -1,4 +1,26 @@
-.carousel .item img {
+.mx-carousel {
```

Comment:
The `mx-` prefix is reserved for core widgets, let's prefix them with `widget-`, so in this case `widget-carousel`.